shell/platform/common/cpp/public/flutter_plugin_registrar.h

```diff
@@ -26,6 +27,10 @@ typedef void (*FlutterDesktopOnRegistrarDestroyed)(
 FLUTTER_EXPORT FlutterDesktopMessengerRef
 FlutterDesktopRegistrarGetMessenger(FlutterDesktopPluginRegistrarRef registrar);
 
+// Returns the texture registrar associated with this registrar.
+FLUTTER_EXPORT FlutterDesktopTextureRegistrarRef
+FlutterDesktopGetTextureRegistrar(FlutterDesktopPluginRegistrarRef registrar);
```

Comment:
This should be called `FlutterDesktopRegistrarGetTextureRegistrar` for consistency with the other plugin registrar methods. Although now that there are two registrars, we should really s/Registrar/PluginRegistrar/ in all these function name prefixes.